packages/expo-modules-core/src/web/CoreModule.ts

```diff
@@ -95,7 +95,7 @@ class SharedObject<TEventsMap extends Record<never, never>>
   implements SharedObjectType
 {
   release(): void {
-    throw new Error('Method not implemented.');
+    // no-op on Web, but subclasses can override it if needed.
```

Comment:
We shouldn't throw from here by default as some methods (especially hooks like `useReleasingSharedObject`) can call this and catching the errors doesn't seem reasonable